protos/game_info.proto

```diff
@@ -0,0 +1,23 @@
+package papan;
+
+message OptionValues {
+  repeated string value = 1;
+}
+
+message GameType {
+  // Game type IDs should be unique across the lifetime of a server
+  uint32 game_type_id = 1;
+  string name = 2;
+  uint32 players_min = 3;
+  uint32 players_max = 4;
+  /* When a player joins a game lobby, the app should download the assets from
```

Comment:
Thinking about it a bit more, this might be a bit more complicated. We probably want to be able to set up teams and stuff. In Belotte for example it's very common to play in 2v2.